testcafe/tests/photos/photos_end_delete_all_data.js

```diff
@@ -51,10 +58,16 @@ test(TEST_DELETE2, async t => {
   console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_DELETE2}`)
   await timelinePage.selectPhotosByName([IMG1, IMG2, IMG3, IMG4])
   //pics are removed, there are no more pictures on  page
-  await timelinePage.deletePhotosFromTimeline(4, true)
+  await timelinePage.deletePhotosFromTimeline({
+    numOfFiles: 4,
+    screenshotPath: 'DeleteImage/delete-4-pic-modal',
+    withMask: maskPhotosCluster
+  })
 
   await t.fixtureCtx.vr.takeScreenshotAndUpload({
-    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE2}-1`
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE2}-1`,
```

Comment:
it feels strange that we have screenshots for cleanup/tear down but maybe it is the easiest way to check that cleanup was done?